vip-helpers/vip-newrelic.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Disable New Relic's browser metrics
+ *
+ * Removes NR's JavaScript for tracking browser metrics, including page load times, Apdex score, and more.
+ *
```

Comment:
Do we prefer disabling this feature (RUM) in code or in the New Relic application settings?